StudentsMVC/README.md

```diff
@@ -0,0 +1,76 @@
+# Students with DB #
```

Comment:
That's quite a cool doc! Perfect! :)

Reply:
I've updated it to correspond to StudentsMVC as the description was for the previous project StudentsDB